lib/stream.js

```diff
@@ -15,8 +15,15 @@ var log = isABib ? bunyan.createLogger({name: 'sierra-bib-retriever'}) : bunyan.
 // send data to kinesis Stream
 exports.streamPoster = (records, streamName) => {
   return writeToStreamsClient(streamName, records)
+          .then(() => {
+            return Promise.resolve({sent: records.length, receieved: records.length})
```

Comment:
Ah good catch @rhernand3z . @jobinthomasnypl I think that may need another look. The resolved `sent` and `received` values will always be the same regardless of what failed and was retried. The way this is written, I believe you don't resolve until everything is sent successfully. And since both `sent` and `received` are set to `records.length`, neither of them communicate anything about failures anyway.

Also, receieved misspelled.